js/classes.js

```diff
@@ -97,7 +97,20 @@ class Projectile {
     this.position.y += this.velocity.y;
   }
 }
-
+  function createMultiShotProjectile(xVelocity) {
+    return  projectiles.push(
```

Comment:
the variable **projectiles** is a global variable should not be part of this function.
This function should **only** return a 'new Projectile' and not push into a global variable.
(Single responsibility principle)

Reply:
a function should have one responsibility/purpose in a program. The purpose of this function should only be create(and return) a new projectile.

Reply:
I think I fixed it. I also figured out what I was doing wrong with trying to use static.

Reply:
Looking good!